### PR TITLE
use `reverse` to replace deprecated `flipdim`

### PR DESCRIPTION
**Pull request by ziyiyin97**
`flipdim` already deprecated so changed it to `reverse`

### CODE REVIEW

**Comment by mloubout** [comment]
joExtend already work for multidimensional arrays, that's what `joKron` is for since it's separable

File: src/joLinearFunctionConstructors/joExtend.jl [hunk 1]
```diff
@@ -187,3 +187,9 @@ function joExtend(n::Integer,pad_type::Symbol;
     return nothing
 end
 
+function joExtend(n::NTuple{N, Integer},pad_type::Symbol;
```
Comment (mloubout) [inline src/joLinearFunctionConstructors/joExtend.jl]: That's not the convention in JOLI, the convention is `joXXX(n1, n2, n3 ....)` no a tuple.

Comment (ziyiyin97) [reply src/joLinearFunctionConstructors/joExtend.jl]: Thanks. `Vararg{Integer, N}` would only be OK if that is the last argument. However, there is a `pad_type` after that. Any suggestion?

**Comment by mloubout** [comment]
Update project.toml

**Comment by rafaelorozco** [comment]
Is there a hold up here I can help with? I ran into this error for the second time in the last few months.

**Comment by mloubout** [comment]
Project.toml update for new release

**Comment by mloubout** [comment]
And if it creates a bug then there should be a test associated with it

**Review by mloubout — APPROVED** [review]

**Pull request merged**